src/web/metrics.rs

```diff
@@ -4,45 +4,67 @@ use iron::prelude::*;
 use iron::status::Status;
 use prometheus::{
     opts, register_counter, register_int_counter, register_int_gauge, Encoder, IntCounter,
-    IntGauge, TextEncoder, __register_gauge,
+    IntGauge, TextEncoder, __register_gauge, register_int_counter_vec, IntCounterVec,
+    __register_counter_vec, histogram_opts, register_histogram_vec, HistogramVec,
```

Comment:
I think I commented on this in another PR and you said prometheus isn't very careful about it's imports in macros - do you know if that would be fixed in the latest release? We have 0.7 and the latest is 0.8.

Reply:
From looking at the source, it looks like some of the internal macros have proper pathing, but the user-facing ones (that we use) do not